src/message.rs

```diff
@@ -786,7 +787,19 @@ impl Message {
     }
 
     /// Returns the size of the file in bytes, if applicable.
+    /// If message is a pre-message, then this returns size of the to be downloaded file.
     pub async fn get_filebytes(&self, context: &Context) -> Result<Option<u64>> {
```

Comment:
maybe modifying this is not the best idea.
A new property/method might be better and easier to understand.

Like when UIs show attachment size of messages for a storage usage UI, then it would always need to check if it is downloaded yet. Could be confusing with thumbnails which also have their own size.